tools/make_patch.c

```diff
@@ -510,19 +513,19 @@ void interpret_command(char *command, const struct symbol *symbol, struct patch
 		if (argc > 0) offset += strtol(argv[0], NULL, 0);
 
 		if (fseek(orig_rom, offset, SEEK_SET) != 0) {
-			fprintf(stderr, "Error: Could not seek to the offset of %s in the original ROM\n", symbol->name);
+			fprintf(stderr, "Error: Could not seek to the offset of %s in the original ROM\n", lastFoundSymbol->name);
```

Comment:
Instead of "last found symbol", consider "current patch".

Reply:
renamed to currentpatch in https://github.com/pret/pokecrystal/pull/882/commits/bf8178e7328da3240535ef9800cdb5163466bcde